src/sentry/static/sentry/app/components/assigneeSelector.jsx

```diff
@@ -107,13 +106,6 @@ const AssigneeSelectorComponent = createReactClass({
   assignableTeams() {
     let group = GroupStore.get(this.props.id);
 
-    // TODO(billyvg): There seems to be a race condition after merging where `group`
-    // is not found in `GroupStore`, show an error toast for now
-    if (!group) {
-      addErrorMessage(t('Error loading teams, please try again.'));
```

Comment:
This error message was weird anyway because it'd happen after merging similar issues, but now this should redirect you to the correct issue. Removing this so we can see if this still throws errors now.